docs/cmd/kn_service_update.md

```diff
@@ -27,6 +27,7 @@ kn service update NAME [flags]
 ### Options
 
 ```
+      --async                    Create service and don't wait for it to become ready.
```

Comment:
Now this should be, Create or update.
https://github.com/knative/client/blob/0b348d7441fcb48f1e94d36b223674ad8251f9c6/pkg/kn/commands/wait_flags.go#L36

Reply:
Ah, right. Let me fix that (maybe using a switch to have different messags for create and update)